scipy/stats/tests/test_distributions.py

```diff
@@ -1064,6 +1064,29 @@ class TestTruncnorm:
     def setup_method(self):
         np.random.seed(1234)
 
+    @pytest.mark.parametrize("a, b, ref",
+    [(0, 100, 0.7257913526447274),
+    (0.6, 0.7, -2.3027610681852573),
+    (1e-06, 2e-06, -13.815510557964274)])
+    def test_entropy(self, a, b, ref):
+        #All reference values were calculated with mpmath:
+        #import mpmath as mp
+        #mp.dps = 50
+        #def entropy_trun(a, b):
```

Comment:
@stefanv this would have caused lint complaints in the past: PEP8 calls for indenting the continuation lines, and comments need a space after the `#`. I've seen minor style issues like this passing CI recently - did we decide to relax these settings?

Reply:
Ruff, when introduced, didn't yet implement all the rules we had before. The idea is to have a code formatter that gets these stylistic issues out the way first, but we don't. That said, @charliemarsh is working on adding those rules, so we can introduce them once they're available.

Reply:
It's coming along. We've added ~26 more of them, trying to knock out a few each day when I can!

Tracking here: https://github.com/charliermarsh/ruff/issues/2402

Reply:
Thanks @charliermarsh!

Here's the list of rules we left behind: https://github.com/scipy/scipy/pull/17812#issuecomment-1404454909

Reply:
OK, thanks. So we will have another big cleanup when those are added?

Reply:
Hopefully only a small cleanup, unless a lot of errors were introduced since we switched to Ruff.